pyracing/response_objects/session_data.py

```diff
@@ -0,0 +1,280 @@
+            self.evttypename = dict['evttypename']
+            self.clubshortname = dict['clubshortname']
+            self.bestquallapnum = dict['bestquallapnum']
+            self.newlicenselevel = dict['newlicenselevel']
+
+
+class RaceLapsAll:
+    def __init__(self, dict):
+        self.details = dict['details']
+        self.startgrid = dict['startgrid']
+        self.lapdata = dict['lapdata']
+
+    class Details:
+        def __init__(self, dict):
+            self.eventDate = dict['eventDate']
+            self.seasonName = dict['seasonName']
+            self.privateSessionName = dict['privateSessionName']
+            self.seasonID = dict['seasonID']
+            self.privateSessionID = dict['privateSessionID']
+            self.seasonShortName = dict['seasonShortName']
+            self.seriesName = dict['seriesName']
+            self.seriesShortName = dict['seriesShortName']
+            self.race_panel_img = dict['race_panel_img']
+            self.eventTypeName = dict['eventTypeName']
+            self.eventType = dict['eventType']
+            self.officialSession = dict['officialSession']
+            self.sessionId = dict['sessionId']
+            self.subSessionId = dict['subSessionId']
+            self.raceWeek = dict['raceWeek']
+            self.nLapsForQualAvg = dict['nLapsForQualAvg']
+            self.nLapsForSoloAvg = dict['nLapsForSoloAvg']
+            self.trackid = dict['trackid']
+            self.eventDateUTCMilliSecs = dict['eventDateUTCMilliSecs']
+            self.driverChanges = dict['driverChanges']
+            self.trackName = dict['trackName']
+            self.trackConfig = dict['trackConfig']
+
+    class StartGrid:
+        def __init__(self, dict):
+            self.custid = dict['custid']
+            self.displayName = dict['displayName']
+            self.helmetPattern = dict['helmetPattern']
+            self.licenseColor = dict['licenseColor']
+            self.helmetColor1 = dict['helmetColor1']
+            self.helmetColor2 = dict['helmetColor2']
+            self.helmetColor3 = dict['helmetColor3']
+            self.startPos = dict['startPos']
+            self.fastestLapNum = dict['fastestLapNum']
+            self.fastestLapTime = dict['fastestLapTime']
+            self.points = dict['points']
+            self.avgLapTime = dict['avgLapTime']
+            self.carnum = dict['carnum']
+            self.numIncidents = dict['numIncidents']
+            self.finishPos = dict['finishPos']
+            self.groupid = dict['groupid']
+            self.friend = dict['friend']
+            self.watch = dict['watch']
+
+    class LapData:
+        def __init__(self, dict):
+            self.lapnum = dict['lapnum']
+            self.custid = dict['custid']
+            self.carnum = dict['carnum']
+            self.sesTime = dict['sesTime']
+            self.flags = dict['flags']
+
+
+class RaceLapsDriver:
+    def __init__(self, dict):
+        self.lapData = dict['lapData']
```

Comment:
Should this be creating an instance of the `LapData` object? I assume that was the intention here

Reply:
Same with the `Header` and `Driver`, the instances aren't being created here

Reply:
Yes, I mentioned in the PR message that the framework for the subclasses are there, but they need to be implemented still. I was not sure of exactly how to do that.

Reply:
I think these objects shouldn't be sub-objects so lets pull them out into objects just in the file, not inside the class and then we can instantiate in another PR